cws-test/src/test/java/jpl/cws/test/WebTestIT.java

```diff
@@ -349,10 +354,10 @@ private void uploadErrorHandlingProcessDefinitionFile() {
 		
 	private void startProcessFromConsole(String procDefKey) {
 		findOnPage(procDefKey);
-		WebElement startIcon = findElById("start_"+procDefKey+"_icon");
+		WebElement startIcon = findElById("pv-test_set_vars");
```

Comment:
Model names shouldn't be hardcoded in functions that can be used in different places.  Please fix this. Thanks!